app/scripts/components/common/page-header.tsx

```diff
@@ -478,25 +489,41 @@ function PageHeader() {
 
 export default PageHeader;
 
-function UserPagesMenu(props: {
+interface DotMenuItem {
+  id: any;
+  menu: string;
+}
+
+function UserPagesDotMenu(props: {
   isMediumDown: boolean;
   onItemClick: () => void;
 }) {
   const { isMediumDown, onItemClick } = props;
 
-  if (!userPages.length) return <>{false}</>;
+  const dotMenuItems = userPages.reduce((menuItems: DotMenuItem[], id: any) => {
+    const page = getOverride(id as any);
```

Comment:
Does it make sense to cast this id as `PageOverrides` ? Hmm... or do we have to do this because it no longer makes sense too manage the extra pages needed for eic or whatever instance in the future as part of `PageOverrides` because the needs are getting too specific?

Whatever the case, at some point in the future.. if we can tackle debt, it would be great to get rid of this component/page overrides structure and this type